docs/getting-started-guides/aws.md

```diff
@@ -51,6 +51,13 @@ Getting started on AWS EC2
 2. Install and configure [AWS Command Line Interface](http://aws.amazon.com/cli)
 3. You need an AWS [instance profile and role](http://docs.aws.amazon.com/IAM/latest/UserGuide/instance-profiles.html) with EC2 full access.
 
+NOTE: This script use the 'default' AWS profile by default.
+You may explicitly set AWS profile to use using the `AWS_DEFAULT_PFROFILE` environament variable:
```

Comment:
If you fix the typos here (_PFROFILE -> _PROFILE, environament -> environment), you might persuade shippable to rerun the test :-)